rafthttp/stream.go

```diff
@@ -38,6 +39,16 @@ const (
 	streamBufSize = 4096
 )
 
+var (
+	errUnsupportedStreamType = fmt.Errorf("unsupported stream type")
+
+	streamSupportMap = map[streamType]*semver.Version{
```

Comment:
struct versionRange {
   min *semver.Version
   max *semver.Version
}

streamSupportMap = map[streamType]versionRange

I find adding a max version map directly is really ugly. :(